### PR TITLE
PMM-6410 added errexit & xtrace for shell steps

### PR DESCRIPTION
**Pull request by Sevlin**
Added **errexit** and **xtrace** for multiline shell steps
```
set -o errexit
set -o xtrace
```

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]

File: pmm/pmm2-server-autobuild.groovy [hunk 1]
```diff
@@ -30,6 +30,9 @@ pipeline {
 
                 git poll: true, branch: GIT_BRANCH, url: 'http://github.com/Percona-Lab/pmm-submodules'
                 sh '''
+                    set -o errexit
+                    set -o xtrace
```
Comment (hors) [inline pmm/pmm2-server-autobuild.groovy:34]: when you add `set -o xtrace` please make sure that you do not show any secrets

Comment (AlekSi) [reply pmm/pmm2-server-autobuild.groovy]: Well, yeah – only on that line. Do we need other `xtrace` lines added in that PR? Is it safe?

**Review by AlekSi — CHANGES_REQUESTED** [review]
https://github.com/Percona-Lab/jenkins-pipelines/pull/540#discussion_r476308270

**Comment by EvgeniyPatlan** [comment]
> [#540 (comment)](https://github.com/Percona-Lab/jenkins-pipelines/pull/540#discussion_r476308270)

@Sevlin please remove new `set xtrace` and left only `set errexit`

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]
`set -o xtrace` please remove everywhere

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by Sevlin** [comment]
@AlekSi @EvgeniyPatlan removed all `xtrace`'es.

**Review by Sudokamikaze — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**